tools/compogen/pkg/gen/schema.go

```diff
@@ -14,6 +14,6 @@ type property struct {
 }
 
 type objectSchema struct {
-	Properties map[string]property `json:"properties" validate:"gt=0,dive"`
+	Properties map[string]property `json:"properties" validate:"dive"`
```

Comment:
I'd rather avoid empty objects in the definitions and keep this validation but if that's not possible I'm fine with loosening the restrictions. Did you encounter any other case with empty object schemas apart from `resource_specification`?

Reply:
Yeah.. Looks like we can add the validation back. It still works.

Reply:
Let me do a quick try

Reply:
The Instill Model connector will encounter some issues with this (I guess it is caused by `oneof`). However, we are also discussing removing the "external mode" for the Instill Model connector. The problem will be resolved if we decide to remove it.